quarkus/reactive-sql/pom.xml

```diff
@@ -12,10 +12,10 @@
     <maven.compiler.target>11</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <quarkus-plugin.version>1.13.6.Final</quarkus-plugin.version>
+    <quarkus-plugin.version>2.0.0.Final</quarkus-plugin.version>
     <quarkus.platform.artifact-id>quarkus-universe-bom</quarkus.platform.artifact-id>
     <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
-    <quarkus.platform.version>1.13.6.Final</quarkus.platform.version>
+    <quarkus.platform.version>2.0.0.Final</quarkus.platform.version>
     <surefire-plugin.version>3.0.0-M5</surefire-plugin.version>
   </properties>
   <dependencyManagement>
```

Comment:
Should this sample use RESTEasy Reactive as well? if so it'll involve some changes in the resource class.

Reply:
Using RESTEasy reactive though you run into a problem later when running the `initdb` because it is a blocking operation and you can't do that in resteasy reactive. I'm not sure how I would fix that. I've reached out to Georgios to see.

I think for now we keep it with regular resteasy. I'll still add comments re: resteasy reactive within the PR for the guide.

Reply:
We'll leave this alone for now and build a new scenario.